components/xsd-fu/templates-java/OMEXMLModelObject.template

```diff
@@ -221,7 +223,8 @@ ${customUpdatePropertyContent[prop.name]}
 		String value_textContent = element.getTextContent();
 		if (value_textContent.trim().length() > 0) {
 {% if prop.type in model.opts.lang.primitive_base_types %}\
```

Comment:
Does this conditional (and the other instances of it) need `&& prop.type == "base64Binary"` so that it will avoid breakage if we add additional types to `primitive_base_types` in the future?